src/veh_interact.cpp

```diff
@@ -2482,6 +2482,7 @@ void act_vehicle_unload_fuel( vehicle* veh ) {
         uimenu smenu;
         smenu.text = _("Remove what?");
         for( auto & fuel : fuels ) {
+            if( fuel == "plut_cell" && veh->fuel_left( fuel )<500 ) continue;
```

Comment:
Please use `PLUTONIUM_CHARGES` instead of a magical number. Also please add spaces around binary operators.